src/common/crypto/mbedtls/cryptoprovider.cpp

```diff
@@ -269,6 +270,25 @@ Error MbedTLSCryptoProvider::PEMToX509Certs(const String& pemBlob, Array<x509::C
     return ErrorEnum::eNone;
 }
 
+#define PEM_BEGIN_CRT "-----BEGIN CERTIFICATE-----\n"
```

Comment:
`PEM_BEGIN_CERT`: cert acronym suits better to certificate.

Reply:
changed